KPI-team1-frontend/src/App.tsx

```diff
@@ -3,14 +3,30 @@ import Sidebar from "./components/Sidebar";
 import { supabase } from "./supabase";
 import { useEffect, useState } from "react";
 import { User } from "./model/user";
+import { HiOutlineMagnifyingGlass } from "react-icons/hi2";
+import { PiBell } from "react-icons/pi";
+import { Circles } from "./model/circle";
 
 const initialUser: User = {
   id: "",
   email: "",
 };
 
+// const initialCircles: Circles = {
+//   circle_name: "",
+//   circle_user:
+// }
+
+// const initialCircleUser: CircleUser = {
+//   circle_user_id: ;
+//   circle_id: number;
+//   user_id: string;
+// }
+
 export default function App() {
   const [user, setUser] = useState<User>(initialUser);
```

Comment:
I wonder if this should be nullable and part of a global context? That way we don't need to pass the user between components too much and when user is null we would simply know that he is not logged in. Also - since if user is not logged in he shouldn't be able to view his circles, maybe we could use some common class for this?

Reply:
I'm not entirely certain that I understand the meaning behind your comment. By the way, if it's possible for you to implement that, could you please assist in adjusting my code?